### PR TITLE
PRM-1749 Updated dependencies + Logging

### PR DESCRIPTION
**Pull request by AndrewRButcher**

### CODE REVIEW

**Comment by AndrewRButcher** [comment]
No issues with Java 11 just used opportunity to update dependencies

File: conf/application-json-logger.xml [hunk 1]
```diff
@@ -6,8 +6,4 @@
     </appender>
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
-
-    <root level="${logger.application:-WARN}">
-        <appender-ref ref="STDOUT"/>
-    </root>
```
Comment (LewisMitchell) [inline conf/application-json-logger.xml:12]: Won't this prevent us from seeing application logs now?

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/application-json-logger.xml [hunk 2]
```diff
@@ -6,8 +6,4 @@
     </appender>
 
     <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
```
[the remark this answers is not in the document]
Comment (LewisMitchell) [reply conf/application-json-logger.xml:8]: Same with logger.application...

**Review by LewisMitchell — CHANGES_REQUESTED** [review]

File: conf/application-json-logger.xml (hunk 1 above)
Comment (AndrewRButcher) [reply conf/application-json-logger.xml]: Reverted

File: conf/application-json-logger.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (AndrewRButcher) [reply conf/application-json-logger.xml]: Just pushed a revision 🤞

**Review by LewisMitchell — APPROVED** [review]

**Pull request merged**